frontend/src/utilities/segmentIOUtils.tsx

```diff
@@ -1,20 +1,26 @@
+import { stringify } from 'yaml';
 import { TrackingEventProperties } from '../types';
+import { DEV_MODE } from './const';
 
 export const fireTrackingEvent = (
   eventType: string,
   properties?: TrackingEventProperties,
 ): void => {
   const clusterID = window.clusterID ?? '';
-  if (window.analytics) {
-    switch (eventType) {
-      case 'identify':
-        window.analytics.identify(properties?.anonymousID, { clusterID });
-        break;
-      case 'page':
-        window.analytics.page(undefined, { clusterID });
-        break;
-      default:
-        window.analytics.track(eventType, { ...properties, clusterID });
+  if (DEV_MODE) {
+    console.log(`Telemetry event triggered: ${eventType} - ${stringify(properties)}`);
```

Comment:
Why are you using yaml stringify? 😕

Reply:
hmm, i don't know if there is a better way to print objects

Reply:
`JSON.stringify(obj, null, 2)` will print the object in a pretty fashion with two spaces for each indent. But honestly you probably don't need to consume multiple lines. Just do `JSON.stringify(obj)` and leave it at that

Reply:
good suggestion, updated